views/lib/footer.hbs

```diff
@@ -19,6 +19,9 @@
                 <span> - </span>
                 <a href="/partner">Partner</a>
             </div>
+            <div>
+                <p>Made with <span style="color:#b10438">❤</span> in Potsdam</p>
```

Comment:
Think about moving the style to the loggedin.scss and give the heart span a class like `color-primary`.